enterprise/backend/src/metabase_enterprise/data_editing/api.clj

```diff
@@ -319,6 +319,43 @@
         provided    (update-keys params #(api/check-400 (param-id (name %)) "Unexpected parameter provided"))]
     (actions/execute-action! action (merge row-params provided))))
 
+(api.macros/defendpoint :get "/tmp-action"
+  [_
+   _
+   _]
+  (api/check-superuser)
+  (let [databases          (t2/select [:model/Database :id :settings])
+        editable-database? #(-> % :settings :database-enable-table-editing boolean)
```

Comment:
Kinda bugs me this isn't comp. You like to read from left to right or something? 😄

Reply:
just as another data point: I love `#(-> % ...)` so much more than comp let it me damned, especially in cases like this, with data structure traversal.

Reply:
Don't send a macro to do a functions job ;-)

Yeah this is pretty subjective and I guess I'm out numbered badly! 😄

Reply:
2bdf5d7d8d9e